examples/README.md

```diff
@@ -0,0 +1,64 @@
+Creating a basic S2I builder image  
+---
+
+### Getting started  
+
+Directory skeleton:  
+
+* `Dockerfile` – standard Dockerfile where we’ll define the base builder image.  
+* `assemble` - responsible for building the application.  
+* `run` - responsible for running the application.  
+* `usage` - optional script responsible for printing the usage of the builder.  
+* `save-artifacts` - optional script for incremental builds that save built artifacts.  
+
+We are going to start by adding our logic to the [`Dockerfile`](nginx-app/Dockerfile).  
+
+In our [`assemble`](nginx-app/assemble) script we define other actions that need to be run when we want to install our application, this is where we build python modules or bundle install our required components, however in this example we leave it as it is. It will only override the default index.html file. It will also copy over any previously saved artifacts.  
+
+Let's create our [`run`](nginx-app/run) script that will start the application.  
+
+We can also specify a logic for an incremental build in [`save-artifacts`](nginx-app/save-artifacts).    
+
+We are now ready to create our builder image that contains nginx and we will name it `nginx-centos7`.
```

Comment:
Need to tell them to run this from within the directory containing the sample pieces referenced above.